happytransformer/happy_word_prediction.py

```diff
@@ -7,9 +7,11 @@
 
 )
 import torch
-
+import collections
 from happytransformer.happy_transformer import HappyTransformer
-from happytransformer.mwp.trainer import  MWPTrainer
+from happytransformer.mwp.trainer import WPTrainer
+
+WPOutput = collections.namedtuple("WPOutput", ["token_str", "score"])
```

Comment:
I think `WordPredictionResult` would be a more descriptive name. "Output" sometimes has broader connotations, whereas Result is more likely to be interpreted by users as a single item in my opinion.

Reply:
Updated